app/models/differential_expression_parameters.rb

```diff
@@ -1,13 +1,14 @@
 # class to hold parameters specific to differential expression jobs in PAPI
 class DifferentialExpressionParameters
   include ActiveModel::Model
+  include Parameterizable
 
   # acceptable Google N1 machine types
   # https://cloud.google.com/compute/docs/general-purpose-machines#n1-high-memory
   GOOGLE_VM_MACHINE_TYPES = [2, 4, 8, 16, 32, 64, 96].map { |i| "n1-highmem-#{i}" }.freeze
 
-  # regular expression to validate GS url format
-  GS_URL_REGEXP = %r{\Ags://}.freeze
+  # name of Python parameter that invokes correct parser
```

Comment:
Clearer, more correct:

```suggestion
  # name of Ingest Pipeline CLI parameter that invokes correct parser
```

Reply:
[81763b0](https://github.com/broadinstitute/single_cell_portal_core/pull/1634/commits/81763b0d5ce57fbb1355cce3b9e39c97052af5c4)